src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4027,40 +4027,40 @@ components:
     ##############################################################################
     SearchIndexRequest:
       required:
-        - sqlQuery
+        - sql
       type: object
       properties:
-        sqlQuery:
+        sql:
           type: string
           description: >
-            SQL to index BigQuery columns into Elasticsearch
+            SQL to index BigQuery columns for searching
```

Comment:
I think this is fine for today but eventually, this will be more than BigQuery (e.g. Synapse on Azure)